qa.hawk/src/main/java/org/aksw/hawk/nlp/SentenceToSequence.java

```diff
@@ -20,6 +20,10 @@
 	public static void runPhraseCombination(final HAWKQuestion q, final List<String> tokens, final Map<String, String> label2pos) {
 		//remove quotation marks from the sentence, then send it to the combination algo
```

Comment:
You changed a lot of things in the file, does the comment still describes what this function is doing?